docs/howto/partitions.rst

```diff
@@ -64,26 +70,31 @@ To add partition support to an application, two basic changes are needed:
       but partitions cannot be enabled via :class:`AppFeatures
       <craft_application.AppFeatures>`!
 
-#. Define the list of partitions
+   .. NOTE::
+      In unit tests, you may need to call this between tests, otherwise the
+      :class:`Features <craft_parts.Features>` global singleton may complain as
+      successive tests try to enable partitions repeatedly.
 
-   We need to tell the :class:`LifecycleManager <craft_parts.LifecycleManager>`
-   class about our partitions, but applications do not usually directly
-   instantiate the LifecycleManager.
+      .. code-block:: python
+
+        Features.reset()
+
+
+
+#. Define the list of partitions
 
-   Instead, override your :class:`Application
-   <craft_application.Application>`'s ``_setup_partitions`` method, and return
-   a list of the partitions, which will eventually be passed to the
-   :class:`LifecycleManager <craft_parts.LifecycleManager>`:
+   Override your :class:`Application <craft_application.Application>`'s
+   ``_setup_partitions`` method and return the list of the partitions.
```

Comment:
```suggestion
   Override the  ``_setup_partitions`` method of your :class:`Application
   <craft_application.Application>` class and return the list of the
   partitions.
```

It's better to avoid adding possessives to compound nouns.

Reply:
Nice